src/web/components/Core/StatusPopup.tsx

```diff
@@ -2,15 +2,21 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { useEffect } from 'react';
```

Comment:
I've noticed that if the user triggers a second Popup before the first one goes away, the second one goes away very quickly because the timer hasn't reset. Could you please add `message` and `status` as dependencies to the timer `useEffect` in this file so the timer resets when the message & status change?

https://github.com/IABTechLab/uid2-self-serve-portal/assets/137864392/629e8511-8de5-4929-a6a7-8e9c2ed49f19

Reply:
In the case of your example, neither status nor message will change since you're deleting twice.  I'll try to think of something else on addition to those.

Reply:
yeah good point, I made that change locally then tested with two different actions 🤦 .  maybe some sort of id or something

Reply:
I'm going to address this in a separate PR.